assets/js/blocks/checkout/index.js

```diff
@@ -13,6 +13,7 @@ import './cart';
 import './checkbox';
 import './coupon';
 import './input';
+import './order-button';
```

Comment:
Looks like this has come back after my push to master, but shouldn't be in this PR – once #19 is merged this will be a conflict/duplicate